protos/local/temporal/sdk/core/workflow_commands/workflow_commands.proto

```diff
@@ -77,6 +77,10 @@ message ScheduleActivity {
     common.RetryPolicy retry_policy = 12;
     /// Defines how the workflow will wait (or not) for cancellation of the activity to be confirmed
     ActivityCancellationType cancellation_type = 13;
+    /// If set, the worker will not tell the service that it can immediately start executing this
+    /// activity. When unset/default, workers will always attempt to do so if activity execution
+    /// slots are available.
+    bool do_not_eagerly_execute = 14;
```

Comment:
Can we make this `eagerly_execute` instead of using the negative form?
Lang can put `true` in here by default.

Reply:
I think it's a better to have it work this way. Lang shouldn't need to explicitly pass something that is the default, it makes it easier for any given lang implementation to make a mistake. More importantly, lang _doesn't have_ the information needed to know if it actually can eagerly execute.

However, it very much can explicitly state when it would _not_ like eager execution to happen.